projects/migrations/0010_auto_20180413_1355.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.10 on 2018-04-13 20:55
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0009_public_metadata_tables_projects_table'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='public_data_tables',
+            name='bq_dataset',
+            field=models.CharField(max_length=100, null=True),
+        ),
+        migrations.AddField(
+            model_name='public_metadata_tables',
+            name='biospec_bq_table',
+            field=models.CharField(max_length=100, null=True),
+        ),
+        migrations.AddField(
+            model_name='public_metadata_tables',
+            name='bq_dataset',
+            field=models.CharField(max_length=100, null=True),
+        ),
+        migrations.AddField(
+            model_name='public_metadata_tables',
+            name='clin_bq_table',
+            field=models.CharField(max_length=100, null=True),
+        ),
+    ]
```

Comment:
Can you delete this one and then re-commit and push? Then I can approve this one.